### PR TITLE
[rum] core web vitals

### PR DESCRIPTION
**Pull request by hdelaby**
### What does this PR do?

* Document Core Web Vitals
* Reorganize the "Monitoring page performance" section: with CWV becoming so central in the product, it becomes less and less important to bring forward other metrics in the docs.

### Motivation

New feature

### Preview

OLD PAGE: https://docs.datadoghq.com/real_user_monitoring/browser/monitoring_page_performance/

NEW PAGE: https://docs-staging.datadoghq.com/hdelaby/core-web-vitals/real_user_monitoring/browser/monitoring_page_performance#core-web-vitals

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 1]
```diff
@@ -16,48 +16,51 @@ further_reading:
...
+| `view.first_input_delay`        | number (ns) | Time elapsed between a user’s first interaction with the page and the browser’s response.                                                                                                                             |
+| `view.cumulative_layout_shift`  | number      | Quantifies unexpected page movement due to dynamically loaded content (for example, third-party ads) where 0 means no shifts happening.                                                                               |
+| `view.loading_time`             | number (ns) | Time until the page is ready and no network request or DOM mutation is currently occurring. [More info](#how-is-loading-time-calculated).                                                                             |
+| `view.first_contentful_paint`   | number (ns) | Time when the browser first renders any text, image (including background images), non-white canvas, or SVG. For more information about browser rendering, see the [w3 definition][11].                               |
```
Comment (bcaudan) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:51]:
```suggestion
| `view.first_contentful_paint`   | number (ns) | Time when the browser first renders any text, image (including background images), non-white canvas, or SVG. For more information about browser rendering, see the [w3c definition][11].                               |
```

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 2]
```diff
@@ -16,48 +16,51 @@ further_reading:
...
-- **First Contentful Paint:** How long until something is displayed. Check the RUM waterfall for blocking resources and long tasks preventing the browser from rendering content.
-- **Loading Time:** How long until the page is interactive. Check the RUM waterfall to see if you're loading too many assets or if some resources are blocking the rendering.
+**Note**: Metrics collected from your real users can differ from those calculated for pages loaded in a synthetic environment.
 
-## All performance metrics
+### All performance metrics
 
-For information about the default attributes for all RUM event types, see [Data Collected][5]. For information about configuring for sampling or global context see [Advanced Configuration][6]. The following table lists Datadog-specific metrics along with performance metrics collected from the [Navigation Timing API][7] and [Paint Timing API][8]:
+RUM view performance metrics are collected from the [Navigation Timing API][8], the [Paint Timing API][9], the [web-vitals library][10] or in-house algorithms.
```
[the remark this answers is not in the document]
Comment (hdelaby) [reply content/en/real_user_monitoring/browser/monitoring_page_performance.md:42]: * Would "in-house implementation of the web-vitals library" be better?
* There are already links to the web.dev docs for each metric. IMO this content would not be understood by 99% of our users (just like Navigation Timing API and similar links on the page to be honest...). What do you think?

[the remark this answers is not in the document]
Comment (bcaudan) [reply content/en/real_user_monitoring/browser/monitoring_page_performance.md:42]: > Would "in-house implementation of the web-vitals library" be better?

Do we care about how we have implemented web vitals?

> There are already links to the web.dev docs for each metric. IMO this content would not be understood by 99% of our users (just like Navigation Timing API and similar links on the page to be honest...). What do you think?

agree, IMO, Links to spec could be only added in metric description.

FMU the intention was to inform that some of the metric are "standard" and other are in house.
If that is right, maybe we could make the distinction clearer by having two metrics table, one for standard browser metrics and another for in house metrics. 
wdyt?

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 3]
```diff
@@ -176,6 +176,9 @@ RUM view performance metrics are collected from both the [Paint Timing API][2] a
...
+| `view.first_input_delay`        | number (ns) | Time elapsed between a user’s first interaction with the page and the browser’s response.                                                                                                                             |
+| `view.cumulative_layout_shift`  | number      | Quantifies unexpected page movement due to dynamically loaded content (for example, third-party ads) where 0 means no shifts happening.                                                                               |
 | `view.loading_time`                             | number (ns) | Time until the page is ready and no network request or DOM mutation is currently occurring. [More info on how view loading time is collected][4]|
 | `view.first_contentful_paint` | number (ns) | Time when the browser first renders any text, image (including background images), non-white canvas, or SVG. For more information about browser rendering, see the [w3 definition][5].                                                                                            |
```
Comment (hdelaby) [inline content/en/real_user_monitoring/browser/data_collected.md:183]:
```suggestion
| `view.first_contentful_paint` | number (ns) | Time when the browser first renders any text, image (including background images), non-white canvas, or SVG. For more information about browser rendering, see the [w3c definition][5].                                                                                            |
```

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (hdelaby) [reply content/en/real_user_monitoring/browser/monitoring_page_performance.md:42]: You're right, that's useless, I'll just delete that sentence altogether. We'll probably remove this list of all metrics and just keep it in the data collected section, so no need for separated tables on this page 🔥

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 4]
```diff
@@ -16,46 +16,47 @@ further_reading:
...
-
-1. Set up and configure [RUM Browser monitoring][1] for your application.
-2. From the RUM Applications page, open the [Performance Overview dashboard][2]. It provides you with a high-level view of your monitored pages. Apply filters to narrow your search and uncover performance problems.
-3. When you find a page with problem, dive from the overview dashboard quickly into a RUM view that exemplifies the problem by clicking **View RUM events** to open it in the [RUM Explorer][3], and then clicking a specific user view. The waterfall in the side panel shows performance details, such as load times for resources, that will help you troubleshoot the bottleneck or other issue.
-    {{< img src="real_user_monitoring/browser/rum-page-performance-dive.gif" alt="Diving from Performance Overview dashboard to performance details for a particular view with an issue"  >}}
+{{< img src="real_user_monitoring/browser/waterfall.png" alt="Real User Monitoring page load waterfall" style="width:75%;" >}}
 
 ## Performance metrics for views
+RUM view events collect extensive performance metrics for every single page view. We recommend analyzing the performance metrics in the following ways:
```
Comment (apigirl) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:22]:
```suggestion

RUM view events collect extensive performance metrics for every single page view. Datadog recommends analyzing the performance metrics in the following ways:
```

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 5]
```diff
@@ -16,46 +16,47 @@ further_reading:
...
+- **Dashboards**: they provide you with a bird-eye view of your application's performance. For example, the out-of-the-box [Performance Overview dashboard][1] can be filtered on [default attributes][2] collected by RUM to surface issues affecting only a subset of users. This dashboard can be cloned and customized to your specific needs. All [RUM performance metrics](#all-performance-metrics) can be used in dashboard queries.
```
Comment (apigirl) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:23]:
```suggestion
- **Dashboards**: provides you with a high-level view of your application's performance. For example, the out-of-the-box [Performance Overview dashboard][1] can be filtered on [default attributes][2] collected by RUM to surface issues impacting a subset of users. This dashboard can be cloned and customized to your specific needs. All [RUM performance metrics](#all-performance-metrics) can be used in dashboard queries.
```

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 6]
```diff
@@ -16,46 +16,47 @@ further_reading:
...
+- **RUM waterfall**: accessible for every single RUM view event in the [RUM Explorer][3], it lets you troubleshoot the performance of a specific page view. It shows how your website assets and resources, long tasks, and frontend errors affect performance for your end users, at the page level.
+
+### Core Web Vitals
 
-Performance metrics are collected for each individual view. These metrics can pinpoint exactly where the problem is occurring when used with the RUM context collected by default (current page view information, GeoIP data, browser information, etc.) and extended with [Global Context][4].
+[Google's Core Web Vitals][4] are a set of three metrics designed to monitor a site's user experience. These metrics, which focus on load performance, interactivity, and visual stability, give you an overview of what matters the most to your users. Each metric comes with guidance on the range of values that translate to good user experience. We recommend monitoring the 75th percentile for these metrics.
```
Comment (apigirl) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:28]:
```suggestion
[Google's Core Web Vitals][4] are a set of three metrics designed to monitor a site's user experience. These metrics focus on giving you a view of load performance, interactivity, and visual stability. Each metric comes with guidance on the range of values that translate to good user experience. Datadog recommends monitoring the 75th percentile for these metrics.
```

**Review by apigirl — COMMENTED** [review]
just some small feedback to fit into Datadog style

**Review by apigirl — APPROVED** [review]

**Pull request merged**